pkg/scheduler/algorithmprovider/defaults/defaults_test.go

```diff
@@ -19,9 +19,15 @@ package defaults
 import (
 	"testing"
 
+	"github.com/google/go-cmp/cmp"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes/fake"
+
```

Comment:
just one block

Reply:
I thought k/k is a separate block from the reset, no? can you put a link to the recommendation on how to organize imports?

Reply:
Truth is we are not consistent

effective go doesn't say anything about it:

https://github.com/golang/go/wiki/CodeReviewComments#imports